articles/cosmos-db/nosql/dynamo-to-cosmos.md

```diff
@@ -48,31 +48,31 @@ The following JSON object represents the data format in DynamoDB:
 
 ```json
 {
-TableName: "Music",
-KeySchema: [
-{ 
-  AttributeName: "Artist",
-  KeyType: "HASH", //Partition key
-},
-{ 
-  AttributeName: "SongTitle",
-  KeyType: "RANGE" //Sort key
-}
-],
-AttributeDefinitions: [
-{ 
-  AttributeName: "Artist",
-  AttributeType: "S"
-},
-{ 
-  AttributeName: "SongTitle",
-  AttributeType: "S"
-}
-],
-ProvisionedThroughput: {
-  ReadCapacityUnits: 1,
-  WriteCapacityUnits: 1
- }
+  "TableName": "Music",
+  "KeySchema": [
+    { 
+      "AttributeName": "Artist",
+      "KeyType": "HASH", //Partition key
```

Comment:
```suggestion
      "KeyType": "HASH",
```

Comments are not allowed in JSON

Reply:
I added the notes from the comments as text after the JSON.